### PR TITLE
feat(billing.termination): Add new reasons

### PR DESCRIPTION
**Pull request by lizardK**
### References
WEBSITE-1998

### CODE REVIEW

File: client/app/account/billing/translations/Messages_fr_FR.xml [hunk 1]
```diff
@@ -1193,14 +1193,18 @@
...
+   <translation id="terminate_service_reason_LACK_OF_PERFORMANCES" qtlid="299975">Les performances du service sont insuffisantes</translation>
    <translation id="terminate_service_reason_NOT_NEEDED_ANYMORE" qtlid="299949">Je n'utilise plus le service</translation>
    <translation id="terminate_service_reason_MIGRATED_TO_ANOTHER_OVH_PRODUCT" qtlid="300014">J'utilise un autre service OVH</translation>
    <translation id="terminate_service_reason_MIGRATED_TO_COMPETITOR" qtlid="300027">J'utilise le service d'un concurrent</translation>
    <translation id="terminate_service_reason_NOT_RELIABLE" qtlid="299988">Le service manque de fiabilité</translation>
    <translation id="terminate_service_reason_OTHER" qtlid="300040">Autres...</translation>
    <translation id="terminate_service_reason_TOO_EXPENSIVE" qtlid="300001">Le service coûte trop cher</translation>
-   <translation id="terminate_service_reason_TOO_HARD_TO_USE" qtlid="299962">Les outils sont trop complexes</translation>
+   <translation id="terminate_service_reason_TOO_HARD_TO_USE" qtlid="299962">Les outils de gestion (espace client, facturation) sont trop complexes</translation>
```
Comment (antleblanc) [inline client/app/account/billing/translations/Messages_fr_FR.xml]: `qtlid="299962"` should be removed right?

**Comment by ovh-cds** [comment]
CDS Report build-yarn#1628.0 ✘  
* Stage 1
  * deps ✔ 
* Stage 2
  * build ✔ 
  * test ✘

File: client/app/account/billing/translations/Messages_fr_FR.xml [hunk 2]
```diff
@@ -1193,14 +1193,18 @@
    <translation id="terminate_service_services_renew" qtlid="168172">Renouvellement</translation>
    <translation id="terminate_service_services_frequency" qtlid="30442">Fréquence</translation>
    <translation id="terminate_service_reason_title" qtlid="440009">Raison de votre résiliation</translation>
-   <translation id="terminate_service_reason_LACK_OF_PERFORMANCES" qtlid="299975">Les performances du systèmes sont insuffisantes</translation>
+   <translation id="terminate_service_reason_LACK_OF_PERFORMANCES" qtlid="299975">Les performances du service sont insuffisantes</translation>
```
Comment (FredericEspiau) [inline client/app/account/billing/translations/Messages_fr_FR.xml]: Shouldn't you delete the `qtlid` ?

File: client/app/account/billing/translations/Messages_fr_FR.xml (hunk 1 above)
Comment (Jisay) [inline client/app/account/billing/translations/Messages_fr_FR.xml]: the qtlid should be removed

File: client/app/account/billing/translations/Messages_fr_FR.xml [hunk 3]
```diff
@@ -1209,8 +1213,8 @@
    <translation id="terminate_service_futureUses_NOT_REPLACING_SERVICE" qtlid="502346">Je ne veux pas remplacer mon service</translation>
    <translation id="terminate_service_futureUses_OTHER" qtlid="300040">Autres...</translation>
    <translation id="terminate_service_futureUses_SUBSCRIBE_AN_OTHER_SERVICE" qtlid="502359">Je veux souscrire à un autre service OVH</translation>
-   <translation id="terminate_service_futureUses_SUBSCRIBE_OTHER_KIND_OF_SERVICE_WITH_COMPETITOR" qtlid="502372">Je veux souscrire à un autre type de service chez un concurrent</translation>
-   <translation id="terminate_service_futureUses_SUBSCRIBE_SIMILAR_SERVICE_WITH_COMPETITOR" qtlid="502385">Je veux souscrire à un service similaire chez un concurrent</translation>
+   <translation id="terminate_service_futureUses_SUBSCRIBE_OTHER_KIND_OF_SERVICE_WITH_COMPETITOR" qtlid="502372">Je veux souscrire à un autre type de service auprès d’un autre fournisseur</translation>
```
Comment (cbourgois) [inline client/app/account/billing/translations/Messages_fr_FR.xml]: some `qtlid` to remove

**Pull request merged**